docs_espressif/en/additionalfeatures/project-configuration.rst

```diff
@@ -0,0 +1,82 @@
+Project Configuration Editor
+====================================
+
+To allow the user to have multiple configurations for the same project, the user can define several settings to produce different build results. For example, take a look at the `Multiple configuration tutorial <multiple_config>`_ which uses the ESP-IDF CMake build system `multi_config <https://github.com/espressif/esp-idf/tree/master/examples/build_system/cmake/multi_config>`_ example.
+
+Use the **ESP-IDF: Open Project Configuration** to manage the project configuration profiles to record the following settings for each configuration:
+
+- **idf.cmakeCompilerArgs**
```

Comment:
I think we can use similar table as the ones we have in the  Settings page here
<img width="752" alt="Screenshot 2024-08-08 at 17 38 57" src="https://github.com/user-attachments/assets/a97d3d08-af61-4241-bc83-1668be52b979">

Reply:
Or even link the Settings page